omnibus_build.sh

```diff
@@ -9,11 +9,15 @@
 PROJECT_DIR=dd-agent-omnibus
 PROJECT_NAME=datadog-agent
 LOG_LEVEL=${LOG_LEVEL:-"info"}
+export AGENT_BRANCH=${AGENT_BRANCH:-"master"}
```

Comment:
I added it, so the user could specify a branch for the local repo if he so wished (ie. a local dev branch).